### ISSUE TITLE
No module named 'pyisy'

### ISSUE THREAD

**Issue by Jefe2** [shbatm/hacs-isy994#60]
I’ve been running v1.4.2 of the ISY994 custom component successfully for a couple days.  Yesterday, I upgraded HA Core from 0.109.3 to 0.109.4.  Now I get this error:

Setup failed for isy994: Unable to import component: No module named 'pyisy'

Any ideas?  I rolled back to 0.109.3 but the error persists.  Running on RPi 3B+ with HassOS.
Thanks in advance.

**Comment by shbatm** [comment]
There's some issues with the PyISY package: https://github.com/automicus/PyISY/issues/90

Hoping the new package is pushed today and I'll push an update.

In the meantime you can upgrade HA to 0.109.4 but you'll have to downgrade hacs-isy994 to 1.4.0 (or change [this line](https://github.com/shbatm/hacs-isy994/blob/2ed2f72c34b04b424d34d9aed79c4e76e03f1bca/custom_components/isy994/manifest.json#L5) to `"requirements": ["PyISY_beta==2.0.0rc6"],`)

**Comment by JCrankWV** [comment]
I was able to get hacs-isy994 v1.4.2 running on HA 0.109.4.
I initially has the same error. "No module named 'pyisy"  I uninstalled the hacs-isy994 integration completely then reinstalled it.

**Comment by shbatm** [comment]
@JCrankWV thanks for the update.  There's about a 50-50 shot on if it installs correctly because the `whl` file has issues.

You can also force an install inside your `venv` or `docker` container with:

```shell
 pip install -e git+https://github.com/automicus/pyisy.git#egg=pyisy==2.0.0
```

Or like I said above, change the manifest.json back to PyISY_beta for the time being.

Hoping the new package is pushed tonight and I'll update HACS and release 1.4.2 from beta.

**Comment by shbatm** [comment]
Fixed in 1.4.3
